mix.exs

```diff
@@ -14,7 +14,7 @@ defmodule Swarm.Mixfile do
   def project do
     [app: :swarm,
      version: "3.0.5",
-     elixir: "~> 1.3",
+     elixir: "~> 1.5",
```

Comment:
Is there a reason we're relying on 1.5 here? I don't like breaking backwards compatibility unless we have a good reason to do so.